File: Chnegyi/形狀/ConsoleApplication5/ConsoleApplication5/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication5
{
    class Circle
    {

        public Double radius;
        public void input()
        {
            Console.WriteLine("輸入半徑:");
            radius = consoletoint();


        }
        public double getarea()
        {
            return radius*radius*3.14;

        }
        private Double consoletoint()
        {

            return Convert.ToDouble(Console.ReadLine());

        }
    }
}

File: LiNNN/物件村題目/src/面積2/Circle.java
package 面積2;

public class Circle extends Shape {

        @Override
        public void Area(){
                System.out.println("請輸入半徑");
                double radius = input.nextDouble();
                
                double area = radius * radius * 3.14;
                
               System.out.println("面積 = " + area); 
               
        }
        
}
